src/main/java/Event.java

```diff
@@ -0,0 +1,48 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Encapsulates the data and behaviour of an Event task.
+ *
+ * @author Huang Zhuoyan, Celeste
+ * @version CS2103T AY24/25 Semester 1, G07
+ */
+public class Event extends Task {
+    private LocalDateTime start;
+    private LocalDateTime end;
+    private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM dd yyyy h:mma");
+
+    /**
+     * Constructs a new Event task with the given name, start date and end date.
+     *
+     * @param name The name of the Deadline task.
+     * @param start The LocalDateTime object containing the start date.
+     * @param end The LocalDateTime object containing the end date.
+     */
+    public Event(String name, LocalDateTime start, LocalDateTime end) {
+        super(name);
+        this.start = start;
+        this.end = end;
+    }
+
+    /**
+     * Formats the deadline of the task according to a standard format.
+     *
+     * @param time The LocalDateTime object to be formatted.
+     * @return The formatted deadline, as a String.
+     */
+    public String deadlineFormatter(LocalDateTime time) {
```

Comment:
Perhaps this could be named
```suggestion
    public String formatDeadline(LocalDateTime time) {
```
instead (to follow the verb way) ?

Reply:
Thanks for pointing it out! I've changed it :D